package.json

```diff
@@ -1,6 +1,6 @@
 {
-    "name": "shader-toy",
-    "displayName": "Shader Toy",
+    "name": "shader-toy-mod",
```

Comment:
Revert this file before a merge. This is another good point for a contribution guide.